app/models/concerns/host_common.rb

```diff
@@ -15,14 +15,14 @@ module HostCommon
   included do
     belongs_to_proxy :puppet_proxy,
       :feature => N_('Puppet'),
-      :label => N_('Puppet Master'),
-      :description => N_('Use this puppet server as an initial Puppet Server or to execute puppet runs'),
+      :label => N_('Puppetserver Proxy'),
```

Comment:
I know I suggested this, but is `Puppet Proxy` perhaps sufficient?

Reply:
Yeah I'll squash it once we get it right - Hmm alone Puppet Proxy would be ambiguous, but within the context of the Puppet CA Proxy I think it's sufficiently clear, and does look/sound better.